app/controllers/wikis_controller.rb

```diff
@@ -48,19 +49,22 @@ def create
   end
 
   def edit
-     @user = current_user
+     #@user = current_user
      #@user = User.find(params[:user_id])
-     @wikis = @user.wikis
-     @wiki = @wikis.find(params[:id])
-     #render edit_user_wiki_path
+     #@wikis = @user.wikis
+     #@wiki = @wikis.find(params[:id])
+     @wikis = Wiki.all
+     @wiki = Wiki.find(params[:id])
   end
```

Comment:
To use pundit in the `edit` action, add this as the final line of `edit`:

```diff
...
     @wiki = Wiki.find(params[:id])
+    authorize @wiki
    end
```
The `authorize` method is provided by Pundit. This new line will call the `WikiPolicy#edit?` method. If the current user is not authorized to edit the wiki, a `Pundit::NotAuthorizedError` will be raised.